### DISCUSSION TITLE
Automating email invites to employees to check out devices?

### DISCUSSION THREAD

**Discussion by XForce23** [snipe/snipe-it#12608] [Q&A]
Hi all, we just started to deploy Snipe IT within our company and we've imported all of our devices from our ABM and MDM servers while also syncing our Snipe to our LDAP.  

Now we face the onerous task of assigning people to their corresponding devices, and we would very much rather not have to cross-reference every device we have manually to do so.  Is there a function within Snipe that can send out emails to employees to have them check out their own devices or to confirm which devices they have?  Or are we stuck having to resort to using manpower to finishing this task?

**Comment by snipe** [comment]
We don’t have that built in, but also, from experience, I can tell you that that wouldn’t work the way you’d hope. People won’t reply, or they’ll give what is definitely not a serial number, or they’ll give you the wrong serial number, etc. In a perfect world, people are responsive and smart and good, but it just generally doesn’t work out that way in real life. Ideally, you’d have your MDM phoning home to Snipe-IT to handle the human error side of things.

**Reply by XForce23** [reply]
Thank you for the detailed response!  I was looking at some scripts and other third party options available, however I didn't see anything stating about matching users to serial numbers/devices.  For example jamf2snipe seems robust and popular, however I didn't find in their documentation that regarding assigning users.

In your experience have you tried or heard of something that would be able to accomplish what we're looking for?
